package.json

```diff
@@ -6,7 +6,7 @@
   },
   "description": "Shared TypeScript definitions for Octokit projects",
   "dependencies": {
-    "@octokit/openapi-types": "^13.11.0"
+    "@octokit/openapi-types-ghec": "^14.0.0"
```

Comment:
sorry [my comment](https://github.com/octokit/types.ts/pull/471#issuecomment-1261233824) was about this change

Reply:
Yes that was my idea, similar to what we do for GHES here: https://github.com/octokit/plugin-enterprise-server.js/blob/master/src/generated/